.devcontainer/install.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+conda_env=$(conda env list | grep radis-env)
+
+echo $conda_env
+
+if [[ $conda_env == *"radis-env"* ]]; then
+    exit 0
+fi
+
+conda env create --file environment.yml --solver=libmamba
```

Comment:
```suggestion
conda env create --file environment.yml
```
--libmamba  not needed anymore as it's the default solver in Anaconda now